src/Components/Publishing/Constants.ts

```diff
@@ -93,50 +92,45 @@ export const getAuthorByline = (authors, isEditoral = true) => {
 }
 
 export const getDate = (date, format: DateFormat = "default") => {
-  const today = moment()
-  const isToday = today.isSame(moment(date), "day")
-  const isThisYear =
-    moment(date).format("YYYY") === moment(today).format("YYYY")
+  const today = DateTime.local()
+  const dateTime = DateTime.fromISO(date).setZone("America/New_York")
+  const isToday = today.hasSame(dateTime, "day")
+  const isThisYear = today.hasSame(dateTime, "year")
+
+  const amPm = dateTime.hour >= 12 ? "pm" : "am"
+  const minutes = dateTime.minute < 10 ? "0" + dateTime.minute : dateTime.minute
+  const monthDay = `${dateTime.monthShort} ${dateTime.day}`
+  const monthDayYear = `${dateTime.monthShort} ${dateTime.day}, ${
+    dateTime.year
+  }`
+  let hour
+  if (dateTime.hour > 12) {
+    hour = dateTime.hour - 12
+  } else if (dateTime.hour === 0) {
+    hour = 12
+  } else {
+    hour = dateTime.hour
+  }
+  const time = `${hour}:${minutes}${amPm}`
```

Comment:
I'll echo the same comment from  https://github.com/artsy/palette/pull/505/files/5c3707b56acd3e0f8ede9a53f9473c6d6041c4da#r292103479

This styling seems simple enough to defer to a format string, rather than constructing it ourselves.

Reply:
Luxon's formatting strings don't provide the display formatting for our styling requirements.

Reply:
Like lowercase am/pm for example.

Reply:
(In the future these sorts of things should be called out in the description.)

Reply:
> Luxon's formatting strings don't provide the display formatting for our styling requirements.
>
> Like lowercase am/pm for example.

Ah, I didn't realize that! Thanks for clarifying :+1: 

I found a related issue and a good response to it here: https://github.com/moment/luxon/issues/224#issuecomment-400176269

Looks like we might be able to use `toLocaleParts` to customize the casing on the meridiem part. No need to block this PR though.

I'm also not sure where the decision was made to remove the space between `minutes` and `amPm`. Could you provide a link to that?

Reply:
All of our time formatting is located in this notion doc https://www.notion.so/artsy/Formatting-Time-f6f4aaeeba414583a207013fec5a9794 @dblandin